executor/hash_table.go

```diff
@@ -112,10 +112,18 @@ func (c *hashRowContainer) ShallowCopy() *hashRowContainer {
 	return &newHRC
 }
 
+// GetMatchedRows get matched rows from probeRow. It can be called
+// in multiple goroutines while each goroutine should keep its own
+// h and buf.
```

Comment:
”h and buf"?

Reply:
This is a copy of GetMatchedRowsAndPtrs.